source/templates/workflows.slim

```diff
@@ -63,8 +63,10 @@
 							header.manage-step
 								.title[ng-if="!workflowsCtrl.renamedStep"]
 									button.rename[ng-bind="workflowsCtrl.selectedStep.displayName()" ng-click="workflowsCtrl.renamedStep = workflowsCtrl.selectedStep"]
-									span.verified[ng-if="workflowsCtrl.selectedStep.isVerified() && !workflowsCtrl.selectedStep.isDeprecated()" trigger-popover="#{ data[:strings][:workflows][:steps][:verified] }"] == svg("step/badge-verified")
-									span.community-created[ng-if="!workflowsCtrl.selectedStep.isVerified() && !workflowsCtrl.selectedStep.isDeprecated()" trigger-popover="#{ data[:strings][:workflows][:steps][:community_created] }"] == svg("step/badge-community_created")
+									span[title="Verified step"]
+										r-icon[ng-if="workflowsCtrl.selectedStep.isVerified()" name="'StepThirdParty'" color="'blue-3'"]
+									span[title="Bitrise step"]
+										r-icon[ng-if="workflowsCtrl.selectedStep.isOfficial()" name="'BitriseCertified'" color="'aqua-3'"]
 									span.deprecated[ng-if="workflowsCtrl.selectedStep.isDeprecated()" trigger-popover="#{ data[:strings][:workflows][:steps][:deprecated] }"] == svg("step/badge-deprecated")
```

Comment:
I know it wasn't part of this but can you port this too to bitkit? 🤔

Reply:
The deprecated icon is not in Bitkit yet unfortunately, I'll create a ticket about it (I think the Design team meant to update it anyways)